packages/lb-components/src/views/MainView/sidebar/PointCloudToolSidebar/index.tsx

```diff
@@ -127,6 +127,7 @@ const BoxTrackIDInput = () => {
     }
 
     updateSelectedBox({ trackID: newTrackID });
+    ptCtx?.topViewInstance?.updatePolygonList(ptCtx?.displayPointCloudList ?? []);
```

Comment:
`displayPointCloudList` just have the part of pointCloudList. You need to update all pointCloudList. The newPointCloudList can be got by the return of `updateSelectedBox`